src/components/LiquidityMining/Create/Steps/RewardAmount/index.tsx

```diff
@@ -73,7 +73,7 @@ export default function RewardsSelection({ rewardsArray, setRewardsArray }: Rewa
   )
 
   const handleLocalUserInput = useCallback(
-    (rawValue, index) => {
+    (rawValue: string | undefined, index: number) => {
```

Comment:
```suggestion
    (rawValue?: string, index: number) => {
```

This should work, right?

Reply:
<img width="801" alt="Screenshot 2022-07-08 at 14 17 22" src="https://user-images.githubusercontent.com/102727631/177990609-86af73b5-7f18-42c6-bcfb-d5b81d5de3dd.png">

If we need to make rawValue optional then need to switch the order. But here you need to either pass a string or pass undefined